src/ZuluSCSI_cdrom.cpp

```diff
@@ -144,41 +144,8 @@ static const uint8_t FullTOC[] =
     0x00, // 43: Frame
     0x00, // 44: Zero
     0x00, // 45: PMIN
-    0x00, // 46: PSEC
+    0x02, // 46: PSEC
     0x00, // 47: PFRAME
-    // b0
```

Comment:
The thought was that older systems might not recognizing the entries, particularly with more complicated TOCs. That said, SCSI2SD code is pretty widely distributed, so that may not be a valid concern.